sdk/purview/azure-purview-administration/azure/purview/administration/metadatapolices/__init__.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf-8
```

Comment:
typo, should be `metadatapolicies`. Not even sure we're having this subfolder though, so may end up not mattering

Reply:
Yes, it's important we rename the folder